Project-3/README.md

```diff
@@ -11,67 +11,466 @@ For any question involving the use of Protege, please be sure to import:
 1. In BFO and RO identify at least one object property for each of a-e that _should have the listed property, but which does not_; argue for your case, using examples. Note: It will be easiest to view the object properties in BFO and RO using Protege. 
 ```
   (a)  Reflexive
+
+  Simultaneous_with
+
+  or,
+
+  The object property, “overlaps”, should have the reflexive characteristic listed.
+  It’s definition states: x overlaps y if and only if there exists some z such that x has part z
+  and z part of y. Reflexivity is the characteristic that expresses the relation that an
+  entity be related to itself. The object property “overlaps” should have reflexive as a listed
+  characteristic since an object shares itself (and parts of itself) with itself. 
+
   (b)  Transitive 
+
+  The object property, “existence ends with”, should have the transitive characteristic listed.
```

Comment:
Very good. We should discuss these and consider having you post on the RO issue tracker: https://github.com/oborel/obo-relations